api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -231,12 +231,13 @@
 
         /**
          * <p>
-         *     If called outside an LRA context the resource method execution
-         *     must then continue outside an LRA context.
+         *     If called outside an LRA context, this is no {@link #LRA_HTTP_CONTEXT_HEADER} at all present,
+         *     the resource method execution must then continue outside an LRA context.
          * </p>
          *
          * <p>
-         *     If called inside an LRA context the method is not executed and a
+         *     If called inside an LRA context, this is {@link #LRA_HTTP_CONTEXT_HEADER} is present and it doesn't
+         *     matter if the LRAId refers to an active, inactive or non-existent LRA, the method is not executed and a
```

Comment:
I wanted to make it clear that if the header is present, with whatever id, the call will fail.

Because previously it was just 'inside LRA context' and you could interpret that only if the header denotes a valid, active LRA you get the HTTP Status 412.

Is this more suited?
> If called inside an LRA context, i.e. {@link #LRA_HTTP_CONTEXT_HEADER} is present referring to an active, inactive or non-existent LRA, the method is not executed and a
     <code>412 Precondition Failed</code> HTTP status code is returned
      to the caller.

Reply:
+1